pkg/templateservicebroker/brokerapi/types.go

```diff
@@ -0,0 +1,146 @@
+package brokerapi
```

Comment:
this seems like it belongs under a more generic path than templateservicebroker (presumably ultimately it will be provided by the brokerSDK though)